src/test.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<h2>JavaScript Promise</h2>
+
+<p id="demo"></p>
+
+<script>
+function myDisplayer(some) {
+  document.getElementById("demo").innerHTML = some;
+}
+
+let myPromise = new Promise(function(myResolve, myReject) {
+  let x = 0;
+
+// some code (try to change x to 5)
+
+  if (x == 0) {
+    myResolve("Okaaaa");
+  } else {
+    myReject("Error");
+  }
+});
+
+myPromise.then(
+  ()=>{ console.log('mmmm')},
+  ()=>{ console.log('hiiii')},
```

Comment:
It would be better to remove the console.log() statements. They are useful for testing but it's better to remove them from the final code.

Reply:
Okay thanks for this I fix this